LeaderboardBackend/Controllers/ModshipsController.cs

```diff
@@ -0,0 +1,77 @@
+using LeaderboardBackend.Models.Entities;
+using LeaderboardBackend.Models.Requests.Modships;
+using LeaderboardBackend.Services;
+using LeaderboardBackend.Controllers.Annotations;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LeaderboardBackend.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class ModshipsController : ControllerBase
+{
+	private readonly ILeaderboardService _leaderboardService;
+	private readonly IModshipService _modshipService;
+	private readonly IUserService _userService;
+
+	public ModshipsController(
+		ILeaderboardService leaderboardService,
+		IModshipService modshipService,
+		IUserService userService
+	)
+	{
+		_leaderboardService = leaderboardService;
+		_modshipService = modshipService;
+		_userService = userService;
+	}
+
+	/// <summary>Gets a Modship.</summary>
+	/// <param name="id">The mod User's ID.</param>
+	/// <response code="200">The Modship.</response>
+	/// <response code="404">If no Modship can be found.</response>
+	[ApiConventionMethod(typeof(Conventions),
+						 nameof(Conventions.Get))]
+	[HttpGet("{id}")]
+	public async Task<ActionResult<Modship>> GetModship(Guid id)
+	{
+		Modship? modship = await _modshipService.GetModship(id);
+
+		if (modship == null)
```

Comment:
Nit: We should switch to using `is null` vs `== null` everywhere. Forgot to mention that. I'm gonna go through and change them where I see them in future PRs.